cdap-app-templates/cdap-etl/cdap-data-pipeline-base/src/main/java/io/cdap/cdap/datapipeline/service/ConnectionHandler.java

```diff
@@ -166,8 +166,11 @@ public void getConnection(HttpServiceRequest request, HttpServiceResponder respo
       contextAccessEnforcer.enforce(new ConnectionEntityId(namespace, ConnectionId.getConnectionId(connection)),
                                     StandardPermission.GET);
       Connection conn = store.getConnection(new ConnectionId(namespaceSummary, connection));
-      metrics.count(Constants.Metrics.Connection.CONNECTION_GET_COUNT, 1);
-      metrics.count(Constants.Metrics.Connection.getConnGetMetric(conn.getConnectionType()), 1);
+      Metrics child = metrics.child(ImmutableMap.of(Constants.Metrics.Tag.SYSTEM_APP_ENTITY_TYPE,
+                                                    Constants.CONNECTION_SERVICE_NAME,
+                                                    Constants.Metrics.Tag.SYSTEM_APP_ENTITY_TYPE_NAME,
+                                                    conn.getConnectionType()));
+      child.count(Constants.Metrics.Connection.CONNECTION_GET_COUNT, 1);
```

Comment:
I would suggest we use a different metric name for the child tags. Normally, for ease of metrics query, user only need to provide the required aggregation to get the metric count. This will return doubled value as count. 
Let's use "connections.type.get.count" or something more natural if you can think of

Reply:
Looks like it won't double count, please ignore this comment